test/runtests.jl

```diff
@@ -1,7 +1,13 @@
-using RoundingIntegers
 using Compat, Base.Test
-
-@test isempty(detect_ambiguities(Base, Core, RoundingIntegers))
+if VERSION < v"0.6.0-pre.alpha.229"
+    using RoundingIntegers
+    @test isempty(detect_ambiguities(Base, Core, RoundingIntegers))
+else
+    ambs0 = detect_ambiguities(Base, Core, Compat)
+    using RoundingIntegers
+    ambs1 = detect_ambiguities(Base, Core, Compat, RoundingIntegers)
+    @test isempty(setdiff(ambs1, ambs0))
```

Comment:
Still has the 2 `cat` ambiguities. Compat doesn't add any, but I wanted to be conservative since there's no reason to fail here due to issues with other packages.